src/OpenColorIO/CMakeLists.txt

```diff
@@ -128,6 +133,20 @@ set(SOURCES
 	ViewTransform.cpp
 )
 
+if(OCIO_ADD_EXTRA_BUILTINS)
+	set(SOURCES_BUILTINS
+		transforms/builtins/ArriCameras.cpp
+		transforms/builtins/CanonCameras.cpp
+		transforms/builtins/PanasonicCameras.cpp
+		transforms/builtins/RedCameras.cpp
+		transforms/builtins/SonyCameras.cpp
+	)
+
+	list(INSERT SOURCES 0 ${SOURCES_BUILTINS})
+else()
+	message(WARNING "Disabling supplemental built-in transforms removes built-in camera transforms.")
```

Comment:
Looks good. I think this should also include a note about OCIO config compatibility to be clear. Something like:

`Disabling supplemental built-in transforms removes built-in camera transforms, limiting OCIO configuration compatibility.`

Reply:
Agree. Done